pkg/logcli/client/client.go

```diff
@@ -283,6 +285,16 @@ func (c *DefaultClient) getHTTPRequestHeader() (http.Header, error) {
 	if c.OrgID != "" {
 		h.Set("X-Scope-OrgID", c.OrgID)
 	}
+	//do not check error here
+	tenantIDs, _ := tenant.TenantIDs(ctx)
+	fakeTenant := len(tenantIDs) == 1 && tenantIDs[0] == "fake"
```

Comment:
I think this should be documented better. Why should the error not be checked?

Reply:
done. thanks
```
// ExtractOrgID gets the org ID from the context.
func ExtractOrgID(ctx context.Context) (string, error) {
	orgID, ok := ctx.Value(orgIDContextKey).(string)
	if !ok {
		return "", ErrNoOrgID
	}
	return orgID, nil
}
```